docs/spec/TEMPLATE.md

```diff
@@ -0,0 +1,67 @@
+# ConceptName
+
+<!--
+Enter an English description of the concept here and a rough explanation of how
+it's mapped to the Kubernetes model.
+-->
+
+ConceptName in CF is mapped to Kubernetes BackingObjects.
```

Comment:
Consider: "Cloud Foundry `ConceptName`s are mapped to Kubernetes `BackingObjects` "

^ domain always follows the subject component, backticks are just a personal style.